apps/web/src/components/(teams)/dialogs/invite-team-member-dialog.tsx

```diff
@@ -70,41 +74,61 @@ export const InviteTeamMembersDialog = ({
   ...props
 }: InviteTeamMembersDialogProps) => {
   const [open, setOpen] = useState(false);
+  const uploadRef = useRef<HTMLInputElement | null>(null);
+  const [invitationType, setInvitationType] = useState('INDIVIDIUAL');
 
   const { toast } = useToast();
 
   const form = useForm<TInviteTeamMembersFormSchema>({
     resolver: zodResolver(ZInviteTeamMembersFormSchema),
     defaultValues: {
-      invitations: [
-        {
-          email: '',
-          role: TeamMemberRole.MEMBER,
-        },
-      ],
+      email: '' || [''],
```

Comment:
Seems it would be easier to always make this an array of emails so we don't have to deal with unions. I assume in the UI you would have an option to "Add another member" within the form.